ui/sections/containers/inspectors/container-editor.reel/container-editor.js

```diff
@@ -2,20 +2,18 @@ var AbstractInspector = require("ui/abstract/abstract-inspector").AbstractInspec
 
 exports.ContainerEditor = AbstractInspector.specialize({
 
-    templateDidLoad: {
+    _inspectorTemplateDidLoad: {
         value: function () {
-            var self = this,
-                blockGateKey = this.constructor.DATA_GATE_BLOCK_KEY;
-
-            this._canDrawGate.setField(blockGateKey, false);
+            var self = this;
 
             Promise.all([
```

Comment:
Te resulting `Promise` should be returned, so that the `AbstractInspector.templateDidLoad` method could handle the drawGate